lib/psulib_blacklight/solr_manager.rb

```diff
@@ -55,6 +55,15 @@ def last_incremented_collection
       collections_with_prefix.max_by { |collection| version_number collection }
     end
 
+    def initialize_collection
```

Comment:
@whereismyjetpack 

would you mind adding a spec for this new method, we will get the coverage higher ;) 

https://github.com/psu-libraries/psulib_blacklight/blob/main/spec/lib/psulib_blacklight/solr_manager_spec.rb

Reply:
I can work on this tomorrow.

Reply:
I'm afraid i might need help on this. I can not, for the life of me, ever wrap my head around rspec. 

Given the following code, the create_collection method does get exersized, i see it's `puts` output in the console, but my setup is completely broken. I'm hoping someone can look at this and point at the exact thing i'm doing wrong. 


```
  describe '#initialize_collection' do
    context 'when collection does not exist' do 
      let(:spy_solr_manager) { instance_spy(described_class) }

      before do 
        stub_request(:get, "#{config_obj.url}/solr/admin/collections?action=LIST")
          .to_return(status: 200, body: '{"responseHeader":{"status":0, "QTime":11}, "collections":[""]}')
      end

      it 'does does not add a new collection' do 
        solr_manager.initialize_collection
        expect(spy_solr_manager.initialize_collection).to have_received(:create_collection)
      end
    end
  end
  ```

  Sorry!

Reply:
are you still blocked on this?

Reply:
@banukutlu  is looking at it for me

Reply:
We worked on this and concluded that spies probably aren't needed @awead thoughts?

Reply:
LGTM! 👍